pkg/sql/catalog/systemschema/system.go

```diff
@@ -98,13 +98,16 @@ CREATE TABLE system.settings (
 	DescIDSequenceSchema = `
 CREATE SEQUENCE system.descriptor_id_seq;`
 
-	TenantsTableSchema = `
+	tenantNameComputeExpr = `crdb_internal.pb_to_json('cockroach.sql.sqlbase.TenantInfo':::STRING, info)->>'name':::STRING`
```

Comment:
Asking purely out of curiosity: isn't `:::STRING` superfluous? I was under the impression that `->>` would cast to string.

Reply:
It is not necessary if you type it in in a SQL client, but we have unit tests that assert that the pretty-printed descriptor uses exactly the same text as was entered in the go source. The pretty-print adds the type annotations. So they have to be there in the input as well.

Reply:
Thanks! TIL